src/web/Dockerfile

```diff
@@ -57,10 +63,27 @@ ENV NEXT_PUBLIC_SUPABASE_URL ${SUPABASE_URL}
 ARG ANON_KEY
 ENV NEXT_PUBLIC_ANON_KEY ${ANON_KEY}
 
-RUN npm install --no-package-lock
+RUN pnpm install
+
+RUN pnpm build
+
+## Telescope docs
+WORKDIR /docusaurus
+
+RUN pnpm install
+
+RUN pnpm build
+
+## Deploy ######################################################################
+FROM nginx:1.20.2-alpine as deploy
+
+WORKDIR /
+
+COPY --from=dependencies /app/out /var/www/data
 
-RUN npm run build
+COPY --from=dependencies /docusaurus/build /usr/share/nginx/html/
 
-FROM nginx:stable-alpine as build
+# Add Healthcheck
+HEALTHCHECK --interval=30s --timeout=30s --start-period=5s --retries=3 \
```

Comment:
This is only for Docusaurus. Do we need a Healthcheck for the Telescope app as well?

Reply:
I don't think we need one for either.  If the main page returns 200, it's healthy.